sysutils/yabai/files/patch-makefile.diff

```diff
@@ -0,0 +1,34 @@
+--- makefile.orig	2021-01-22 17:41:47.000000000 -0800
++++ makefile	2021-01-22 17:55:53.000000000 -0800
+@@ -1,6 +1,6 @@
+ FRAMEWORK_PATH = -F/System/Library/PrivateFrameworks
+ FRAMEWORK      = -framework Carbon -framework Cocoa -framework CoreServices -framework SkyLight -framework ScriptingBridge
+-BUILD_FLAGS    = -std=c99 -Wall -g -O0 -fvisibility=hidden -mmacosx-version-min=10.13
++CFLAGS         += -std=c99 -Wall -DNDEBUG -O2 -fvisibility=hidden -mmacosx-version-min=10.13
```

Comment:
you need at least to get rid of `-O2` and `-mmacosx-version-min=10.13` here

Reply:
Sure thing